cdap-docs/integrations/source/index.rst

```diff
@@ -29,8 +29,14 @@ Integrations
 - |cloudera-ingesting|_ Ingesting and exploring data using **Impala**
 
 
+.. |cloudera-navigator| replace:: **Navigator Integration:**
+.. _cloudera-navigator: partners/cloudera/navigator-integration.html
+
+- |cloudera-navigator|_ Metadata search and discovery of CDAP entities from **Cloudera Navigator**
```

Comment:
Repeating my comment since it may have been lost:

"search" and "discovery" are essentially the same thing? Maybe 
...: Metadata-based discovery of CDAP entities from Cloudera Manager